drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -187,10 +188,17 @@ static gnrc_pktsnip_t *_make_netif_hdr(uint8_t *mhr)
     hdr = (gnrc_netif_hdr_t *)snip->data;
     gnrc_netif_hdr_init(hdr, src_len, dst_len);
     if (dst_len > 0) {
+        broadcast = true;
```

Comment:
Not that this code is optimised by any means, but as a proposal, you could set the flag here and remove it below if not applicable. Saves us one byte on stack, one assignment and one if-statement. \o/ 😊